packages/fast-usdc/src/cli/lp-commands.js

```diff
@@ -97,7 +97,6 @@ export const addLPCommands = (
       const usdcAmount = parseUSDCAmount(opts.amount, usdc);
 
       /** @type {import('../types.js').PoolMetrics} */
-      // @ts-expect-error it treats this as "unknown"
```

Comment:
do you want to get rid of the type annotation while you're at it?

Reply:
oh yeah thanks